src/game_frontend/src/pages/Dashboard/layout.tsx

```diff
@@ -11,6 +11,9 @@ const Layout = () => {
       className="exchange"
       style={{
         backgroundImage: 'url("/assets/images/space-exchange-bg.webp")',
+        width: '100vw',           // Full viewport width
+        height: '100vh',         // Full viewport height
+        overflow: 'hidden',     // Ensures no extra space
```

Comment:
Is there a reason for `overflow : hidden`

Reply:
Not  using  ```overflow : hidden```  scroll bar will visible

Reply:
![ScreenRecording2024-12-07at1 24 46PM-ezgif com-video-to-gif-converter](https://github.com/user-attachments/assets/c29eb6b4-e543-4bbc-8c8c-ade1b00e0a29)